src/codesy.coffee

```diff
@@ -40,48 +41,47 @@ codesy.events.submit = (e)->
         codesy.newpage()
     .fail (err)->
       console.log 'codesy: bid update failed' 
-      console.log err  
+      console.dir err
+      console.dir codesy  
   false
 
-codesy.appendForm = (form_html) ->
+codesy.append = (form_html) ->
   $("body").append form_html
   if $("#codesy_bid_form").length > 0
     $("#codesy_bid_form").submit codesy.events.submit
   
-codesy.setDomain = (domain) ->
-  console.log "codesy: new domain is " + domain.domain
-  codesy.auth_token = domain.token
-  codesy.url = domain.domain
-  codesy.newpage()
-
 codesy.newpage = ()->
   $("#codesy_bid").remove()
   if codesy.isIssue window.location.href
     console.log 'codesy: needs bid form'
     console.time "codesy: request form"
-    chrome.storage.local.get (data)->
-      if data.domains[0].domain isnt codesy.url
-        codesy.setDomain(data.domains[0])
-        codesy.bid.get {url:window.location.href}
-          .done (data) ->
-            console.timeEnd "codesy: request form"
-            console.log "codesy: form request success"
-            codesy.appendForm data
-          .fail (err) ->
-            console.timeEnd "codesy: request form"
-            console.log "codesy: form request failed"
-            if err.status = 401
-              codesy.appendForm err.responseText
-            else
-              console.log err
+    codesy.bid.get {url:window.location.href}
```

Comment:
removed the local storage check;  it now runs separately and then fires new page.